virttest/qemu_storage.py

```diff
@@ -13,6 +13,12 @@
 import string
 
 import six
+
+try:
+    from collections.abc import Mapping
+except ImportError:
+    from collections import Mapping
```

Comment:
Seems like https://github.com/avocado-framework/avocado-vt/blob/48b5b3dde0a4506031588f6685db67365fd33839/virttest/qemu_storage.py#L8
is useless, we could delete it.

Reply:
Hi @hellohellenmao and @YongxueHong , actually, we use collections.OrderedDict when getting image opts

Reply:
Updated, thanks!

Reply:
> Hi @hellohellenmao and @YongxueHong , actually, we use collections.OrderedDict when getting image opts

Thanks, I got it, I missed it.